CHANGELOG.md

```diff
@@ -32,6 +32,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Bump maxVideos limit to 25
+- Control video downlink policy with network event response and recovery delays using            
```

Comment:
This should be in `Added`.  Also should mention the name of classes for a bit more context, and link the section in documentation.